basic/pom.xml

```diff
@@ -66,7 +66,7 @@
 				<artifactId>spring-boot-maven-plugin</artifactId>
 				<version>${spring-boot-dependencies.version}</version>
 				<configuration>
-					<mainClass>com.c8y.ms.templates.basic.App</mainClass>
```

Comment:
wrong file. This is the pom for the basic microservice which you have updated

Reply:
i restored complete basic module from main branch